### PR TITLE
Add contribution guide scripts

### PR DESCRIPTION
**Pull request by stefvhuynh**
Addresses https://github.com/FormidableLabs/react-game-kit/issues/46

### CODE REVIEW

**Comment by stefvhuynh** [comment]
NOTE: Need to fix some lint errors before merge.

**Comment by ryan-roemer** [comment]
@stefvhuynh -- Can you also add a `.travis.yml`? I've enabled travis on this repo.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+
+node_js:
+  - "4"
+  - "6"
```
Comment (ryan-roemer) [inline .travis.yml]: Add `"8"`

File: src/components/body.js [hunk 2]
```diff
@@ -79,14 +79,14 @@ export default class Body extends Component {
   constructor(props, context) {
     super(props);
 
-    const { args, children, shape, ...options } = props;
```
Comment (ryan-roemer) [inline src/components/body.js]: The weird thing with object spread is sometimes, folks use this to get **rid** of `children` for the resulting `options` variable. We should check that it is OK for `options.children` to now be added.

File: src/components/body.js [hunk 3]
```diff
@@ -79,14 +79,14 @@ export default class Body extends Component {
...
+    const { args, shape, ...options } = props;
 
     this.body = Bodies[shape](...args, options);
     World.addBody(context.engine.world, this.body);
   }
 
   componentWillReceiveProps(nextProps) {
-    const { args, children, shape, ...options } = nextProps;
```
Comment (ryan-roemer) [inline src/components/body.js]: Same here.

File: src/native/components/body.js [hunk 4]
```diff
@@ -79,14 +79,14 @@ export default class Body extends Component {
   constructor(props, context) {
     super(props);
 
-    const { args, children, shape, ...options } = props;
```
Comment (ryan-roemer) [inline src/native/components/body.js]: And here

File: src/native/components/body.js [hunk 5]
```diff
@@ -79,14 +79,14 @@ export default class Body extends Component {
...
+    const { args, shape, ...options } = props;
 
     this.body = Bodies[shape](...args, options);
     World.addBody(context.engine.world, this.body);
   }
 
   componentWillReceiveProps(nextProps) {
-    const { args, children, shape, ...options } = nextProps;
```
Comment (ryan-roemer) [inline src/native/components/body.js]: And here

File: src/components/body.js (hunk 2 above)
Comment (stefvhuynh) [reply src/components/body.js]: So it looks like these options are passed into a constructor in `matter.js`. It looks like it's fine if `children` is left in, but I think it's better for maintenance to keep it clean and not pass in superfluous arguments. I'll revert these object spread and add an `eslint-disable`.

File: package.json [hunk 6]
```diff
@@ -48,9 +52,10 @@
     "postcss-loader": "^2.0.6",
     "prop-types": "^15.5.10",
     "raw-loader": "^0.5.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "16.0.0",
```
Comment (ryan-roemer) [inline package.json]: Do we need to pin these for RN or something?

Comment (stefvhuynh) [reply package.json]: Yeah, `react-native` doesn't accept a range on `react` as a peer dep. Each version of `react-native` seems to be tied to a specific version of `react`.

Comment (ryan-roemer) [reply package.json]: SGTM

**Pull request merged**